modules/web/tools/gen-default-nodes/gen-default-nodes.js

```diff
@@ -58,7 +58,8 @@ function getConfigs() {
  */
 function generateNodes() {
     getConfigs().then((configs) => {
-        const { endpoint } = configs.services.fragmentParser;
+        const parserEndpoint = configs.services['ballerina-parser'].endpoint;
```

Comment:
do we need the "parserEndpoint" variable?

Reply:
It's not mandatory. But I think it makes it easier to understand the code.